mbtools/validate_mbz_html.py

```diff
@@ -42,7 +42,11 @@
     "https://www.sfusdmath.org",
     "https://www.engageny.org",
     "https://www.khanacademy.org",
-    "https://teacher.desmos.com"
+    "https://teacher.desmos.com",
+    "https://help.desmos.com",
+    "https://student.desmos.com",
+    "https://desmos.com/calculator",
+    "https://geogebra.org/graphing"
```

Comment:
@mjoss17 - Let's not add these to the `VALID_HREF_PREFIXES` as I don't think we need / want to be that flexible. Instead let's put them into `VALID_HREF_VALUES` (e.g. so `https://desmos.com/calculator` is allowed but not `https://desmos.com/calculator/someslug`)